src/back/auth/src/service/session.hpp

```diff
@@ -19,11 +19,25 @@ class Session final {
 	model::Session Create(
 		const OIDCTokens& tokens,
 		const TokenPayload& data,
-		const std::string& userAgent);
+		const std::string& userAgent,
+		const std::chrono::system_clock::time_point& exp);
+
+	model::Session Refresh(
+		const OIDCTokens& tokens,
+		const TokenPayload& data,
+		const std::string& userAgent,
+		const std::chrono::system_clock::time_point& exp,
+		const boost::uuids::uuid& oldSessionId);
 
 private:
 	table::Session& _table;
 	tokens::Session& _tokenizer;
+
+	model::Session prepare(
+		const OIDCTokens& tokens,
+		const TokenPayload& data,
+		const std::string& userAgent,
+		const std::chrono::system_clock::time_point& exp);
```

Comment:
Перенеси пожалуйста объявление функции над объявлением членов класса.
```suggestion
	model::Session prepare(
		const OIDCTokens& tokens,
		const TokenPayload& data,
		const std::string& userAgent,
		const std::chrono::system_clock::time_point& exp);

	table::Session& _table;
	tokens::Session& _tokenizer;
```

Reply:
Готово.